src/screens/Dashboard/TransactionList.tsx

```diff
@@ -62,7 +62,11 @@ class TransactionList extends PureComponent<Props> {
   renderItem = ({ item: transaction }: { item: Transaction }) => {
     return (
       <View style={styles.itemWrapper}>
-        <TransactionItem item={transaction} onPress={this.onTransactionItemPress} />
+        <TransactionItem
+          testID={transaction.note && `transaction-item-${transaction.note}`}
```

Comment:
why note? imho should be testID={`transaction-item-${transaction.txid}`}

Reply:
what exactly is `txid`?

Reply:
@KedziaPawel ?

Reply:
sorry didn't see the comment:D https://electrumx.readthedocs.io/en/latest/protocol-methods.html#blockchain-transaction-get txid is the hash of the transaction from electrum-x

Reply:
We decided to use notes because it faster to verify via UI when we add some unique text inside the note.

Reply:
yes, but it also makes testID specific and impossible to use when the will need to click the transaction without the `note` in another e2e test. Txid is a more generic way

Reply:
Probably we'll switch to `txid` in the future, but right now it'd be problematic to gather the transaction hash of the previously made transaction.

But for sure it's a good improvement and we should follow this approach. Simply not now :)

Reply:
I see, indeed it could be not so simple to get the txid of the transaction created during the test